### ISSUE TITLE
exif string doesn't support UTF-8

### ISSUE THREAD

**Issue by fisherwei** [gomfunkel/node-exif#65]
I am trying to do a patch, it seems worked(in my app).

FYR:

Buffer.js
```
const StringDecoder = require('string_decoder').StringDecoder;
const decoder = new StringDecoder('utf8');

Buffer.prototype.getString = function (offset, length) {
  var string = [];
  for (var i = offset; i < offset + length; i++) {
    string.push(this[i]);
  }
  return decoder.write(Buffer.from(string));
};
```

**Comment by RandScullard** [comment]
I think I have an even simpler solution that works perfectly in my testing:

    Buffer.prototype.getString = function (offset, length) {
      return this.slice(offset, offset + length - 1).toString();
    };

**Comment by oeuillot** [comment]
I have merged the @RandScullard  proposal to version 0.6.1

**Comment by Djiit** [comment]
Hey @oeuillot do you know when the 0.6.1 will land ?
